src/types/general.ts

```diff
@@ -34,11 +34,19 @@ type ServiceConfig = {
   dns_gd: ProviderService
 }
 
+type NameServer = {
```

Comment:
this is godaddy specific, should not be in general types.

Reply:
because if every domain provider created their own types, it'll get messy.

Reply:
Sure make sense